request.go

```diff
@@ -52,16 +52,16 @@ func requestFromPacket(pkt hasPath) Request {
 		request.Flags = p.Flags
 		request.Attrs = p.Attrs.([]byte)
 	case *sshFxpRenamePacket:
-		request.Target = filepath.Clean(p.Newpath)
+		request.Target = filepath.ToSlash(filepath.Clean(p.Newpath))
 	case *sshFxpSymlinkPacket:
-		request.Target = filepath.Clean(p.Linkpath)
+		request.Target = filepath.ToSlash(filepath.Clean(p.Linkpath))
 	}
 	return request
 }
 
 // NewRequest creates a new Request object.
 func NewRequest(method, path string) Request {
-	request := Request{Method: method, Filepath: filepath.Clean(path)}
+	request := Request{Method: method, Filepath: filepath.ToSlash(filepath.Clean(path))}
```

Comment:
This filepath.ToSlash(filepath.Clean() logic appears too many times.

Please break it into a helper function and add a unit test for all the appropriate cases; "/", "//", "\\", "/a/", "\a/", etc